treeherder/webapp/api/push.py

```diff
@@ -70,6 +73,15 @@ def list(self, request, project):
                 )
 
             pushes = pushes.filter(repository=repository)
+    
+        if search_query:
+            search = SearchQuery(search_query, config="english")
```

Comment:
for readability, I'd rename this `search` to `search_query` (after the other `search_query` is renamed to `search_param`)

Reply:
I guess you could even use it directly in the filter below and remove the variable. But that's up to you